### PR TITLE
generate a more readable default label for templated modules

### PR DESCRIPTION
**Pull request by fwyzard**

### CODE REVIEW

**Comment by fwyzard** [comment]
@perrotta , @Martin-Grunewald , this shouldn't break anything, since as far as I know we don't use the autogenerated cfi's explicitly anyway.

@Dr15Jones , maybe what I put in `HLTrigger/HLTcore/interface/DefaultModuleLabel.h` could go in `FWCore/ParameterSet`, either as a free function in a standalone file, or inside `ConfigurationDescription` ?

File: HLTrigger/Egamma/src/HLTElectronPFMTFilter.cc [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #include "HLTrigger/Egamma/interface/HLTElectronPFMTFilter.h"
+#include "HLTrigger/HLTcore/interface/defaultModuleLabel.h"
```
Comment (perrotta) [inline HLTrigger/Egamma/src/HLTElectronPFMTFilter.cc]: defaultModuleLabel.h -> DefaultModuleLabel.h
(and the same in all other #include's)

**Comment by perrotta** [comment]
-1
Please either rename HLTrigger/HLTcore/interface/DefaultModuleLabel.h as HLTrigger/HLTcore/interface/defaultModuleLabel.h or put the capital initial letter in all includes

**Comment by Dr15Jones** [comment]
Given the function starts with a lower case, the file name should also start with a lower case.

**Comment by fwyzard** [comment]
Header file appropriately renamed.

**Comment by perrotta** [comment]
+1

**Pull request merged**